src/lib/y2network/widgets/interfaces_table.rb

```diff
@@ -143,11 +142,15 @@ def create_description
         summary.new(value, config).text
       end
 
-      # Returns a friendly name for a given interface
+      # Returns the connection description if given or the interface friendly name if not
       #
       # @param interface [Interface] Network interface
-      # @return [String] Friendly name for the interface (description or name)
-      def friendly_name(interface)
+      # @param conn [ConnectionConfig::Base] Connection configuration
+      # @return [String] Connection description if given or the friendly name for the interface (
+      #   description or name) if not
+      def description_for(interface, conn)
+        return conn.description unless conn&.description.to_s.empty?
```

Comment:
in theory we can face a problem that someone might use empty name intentionally not to display anything in the overview .... but it sounds a little bit pathologically ;-)

So what about some test(s)?

Reply:
It is not possible from the config file and also not from the **GUI** after the last changes, but wanted to prevent it was initialized by any builder or step because of a leftover.

The not set should cover the case but I can initialize the connection description to empty if you want :)